src/tests/cmocka/test_negcache.c

```diff
@@ -1282,21 +1283,29 @@ static void test_sss_ncache_locate_uid_gid(void **state)
     assert_int_equal(ret, ENOENT);
     ret = sss_ncache_check_locate_uid(ts->ctx, dom, uid);
     assert_int_equal(ret, ENOENT);
+    ret = sss_ncache_check_locate_sid(ts->ctx, dom, sid);
+    assert_int_equal(ret, ENOENT);
 
     ret = sss_ncache_set_locate_gid(ts->ctx, dom, gid);
     assert_int_equal(ret, EOK);
     ret = sss_ncache_set_locate_uid(ts->ctx, dom, uid);
     assert_int_equal(ret, EOK);
+    ret = sss_ncache_set_locate_sid(ts->ctx, dom, sid);
+    assert_int_equal(ret, EOK);
 
     ret = sss_ncache_check_locate_gid(ts->ctx, dom, gid);
     assert_int_equal(ret, EEXIST);
     ret = sss_ncache_check_locate_uid(ts->ctx, dom, uid);
     assert_int_equal(ret, EEXIST);
+    ret = sss_ncache_check_locate_sid(ts->ctx, dom, sid);
+    assert_int_equal(ret, EEXIST);
 
     ret = sss_ncache_check_locate_gid(ts->ctx, dom2, gid);
     assert_int_equal(ret, ENOENT);
     ret = sss_ncache_check_locate_uid(ts->ctx, dom2, uid);
     assert_int_equal(ret, ENOENT);
+    ret = sss_ncache_check_locate_uid(ts->ctx, dom2, sid);
```

Comment:
Hi,

thanks for the patches, they are working well in my tests, but I think there is a typo and you have meant `sss_ncache_check_locate_sid()`.

bye,
Sumit

Reply:
Yes, you are right. Fixed and rebased.